File: mario.c
#include <stdio.h>
#include <cs50.h>

int main (void) {
    int height, row, column, space;
    do {
      height = get_int ("Insira a altura da pirâmide: ");  
    } while(height < 1 || height > 8);
    
    for (row = 0 ; row < height ; row ++){
        
        for (space = 0; space < height - row - 1; space ++){
            printf(" ");
        }
        
        for(column = 0 ; column <= row ; column ++){
            printf("#");
        }
        printf("\n");
    }
    
}

File: Linguagem-Python/sentimental-mario-less/mario.py
# importando função get_int da biblioteca cs50
from cs50 import get_int

# Loop de verificação da altura
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

# Usando for para estruturação da pirâmide
for i in range(0, n, 1):
    for j in range(0, n, 1):
        if (i + j < n - 1):
            print(" ", end="")
        else:
            print("#", end="")
    print()